src/openrct2/interface/Screenshot.cpp

```diff
@@ -196,8 +196,8 @@ std::string screenshot_dump_png_32bpp(int32_t width, int32_t height, const void*
         image.Depth = 32;
         image.Stride = width * 4;
         image.Pixels = std::vector<uint8_t>(pixels8, pixels8 + pixelsLen);
-        Imaging::WriteToFile(path->c_str(), image, IMAGE_FORMAT::PNG_32);
-        return *path;
+        Imaging::WriteToFile(path.value(), image, IMAGE_FORMAT::PNG_32);
```

Comment:
```suggestion
        Imaging::WriteToFile(path.value().c_str(), image, IMAGE_FORMAT::PNG_32);
```

Reply:
The argument type is std::string_view so I don't think c_str() should be used here.

Reply:
Ah, I thought that might have been the case, wanted to make sure the change was intentional.

Reply:
I see, I think it would be a compiler error to convert std::string to const char* but good thinking.